tests/test_cli.py

```diff
@@ -61,6 +61,51 @@ def test_ping(runner):
     assert "Name: docker-cluster" in result.stdout
 
 
+def test_create_index_neither_name_nor_source_passed(runner):
+    result = runner.invoke(main, ["create"])
+    assert result.exit_code == 2
+    assert "Must provide either a name or source for the new index." in result.stdout
+
+
+def test_create_index_name_and_source_passed(runner):
+    result = runner.invoke(
+        main,
+        ["create", "--index", "aspace-2022-09-01t12-34-56", "--source", "aspace"],
+    )
+    assert result.exit_code == 2
+    assert "Must provide either a name or source for the new index." in result.stdout
+
+
+def test_create_index_invalid_name_passed(runner):
+    result = runner.invoke(main, ["create", "--index", "wrong"])
+    assert result.exit_code == 2
+
```

Comment:
Yeah good question, but the check for that doesn't happen here because the output message text from validating these two options happens in the validation function, which is unit tested elsewhere (in the helpers module). We don't even log the message in this module because the click option validation callback happens before this click function runs, so from a logical standpoint the check for that message text doesn't make sense here.